src/sentry/workflow_engine/processors/delayed_workflow.py

```diff
@@ -793,6 +793,31 @@ def _summarize_by_first[T1, T2: int | str](it: Iterable[tuple[T1, T2]]) -> dict[
     return {key: sorted(values) for key, values in result.items()}
 
 
+@instrumented_task(
```

Comment:
🤔 could we move this into the `workflow_engine/tasks` directory so it's a bit easier to find this task code?